dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -234,7 +236,7 @@ public void doImport(PrintWriter out) throws IOException {
         boolean hasAssetDir = false;
         for (int i = 0; i < _tempFiles.length; i++) {
             try {
-				HibernateUtil.closeSession();
+				HibernateUtil.closeSession(); // todo: why this?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)